experimental/packages/opentelemetry-sdk-metrics-base/src/exemplar/SimpleFixedSizeExemplarReservoir.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Context, HrTime } from '@opentelemetry/api';
+import { ValueType, Attributes } from '@opentelemetry/api-metrics';
+import { FixedSizeExemplarReservoirBase } from './ExemplarReservoir';
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+/**
+ * Fixed size reservoir that uses equivalent of naive reservoir sampling
+ * algorithm to accept measurements.
+ * 
+ */
+export class SimpleFixedSizeExemplarReservoir extends FixedSizeExemplarReservoirBase {
+  private _counter: number;
+  constructor(size: number) {
+    super(size);
+    this._counter = 0;
+  }
+
+  getRandomInt(min: number, max: number) { //[min, max)
```

Comment:
`getRandomInt` looks like constantly returning the value `min` as line 38 is equivalent to `Math.floor(Math.random() + min)`, i.e. `Math.floor(min)`.

Reply:
I tested this locally and the findBucketIndex always returns an incremented index. Could you elaborate on why we need `getRandomInt` here?

Reply:
This is based on the naive reservoir sampling algorithm described [here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#exemplar-defaults).

Reply:
Right, noticed you've updated the usage below, then it makes sense.

Reply:
yes, we needed the random integer but your incremented index observation was good catch.